changelog/fix_a_false_negative_for_methods_that_take_a_block_in_void_context copy.md

```diff
@@ -0,0 +1 @@
+* [#11426](https://github.com/rubocop/rubocop/pull/11426): Update `Lint/Void` to suggest `each` in place of `map` or `collect` in void context. ([@krishanbhasin-shopify][])
```

Comment:
Please remove this changelog entry. Because message-only adjustments have no direct behavioral impact.